internal/cri/server/fuzz.go

```diff
@@ -1,33 +0,0 @@
-//go:build gofuzz
```

Comment:
Given we now moved all fuzz tests to go native fuzz and removed `gofuzz` test, I'm not sure how to expose this to fuzz tests in `contrib/fuzz` other than deletion.

We only fuzz test `sandboxstore.Add` directly in fuzz test, which should also be covered by `runPodSandboxFuzz` indirectly.